dev/com.ibm.ws.kernel.boot_fat/publish/files/versionlessServer.xml

```diff
@@ -16,7 +16,7 @@
     
     <featureManager>
         <platform>jakartaee-10.0</platform>
-        <platform>microProfile-4.0</platform>
+        <platform>microProfile-5.0</platform>
```

Comment:
These still conflict.  JEE 10 goes with MP 6 and 6.1.  JEE 9 goes with MP 5.

Reply:
fixed